packages/admin/src/targetGroups/addContacts/AddContactsGridSelect.gql.ts

```diff
@@ -0,0 +1,50 @@
+import { gql } from "@apollo/client";
+
+const targetGroupBrevoContactsFragment = gql`
+    fragment TargetGroupBrevoContactsList on BrevoContact {
+        id
+        createdAt
+        modifiedAt
+        email
+        emailBlacklisted
+        smsBlacklisted
+    }
+`;
+
+export const assignedBrevoContactsQuery = gql`
+    query BrevoContactsGrid(
+        $offset: Int
+        $limit: Int
+        $email: String
+        $scope: EmailCampaignContentScopeInput!
+        $targetGroupId: ID
+        $onlyShowAssignedContactsOfTargetGroup: Boolean
```

Comment:
I'm unsure if I like this `onlyShowAssignedContactsOfTargetGroup` filter. It could also be two queries since the filter can't be set dynamically. 

But it's only a "slight objection"

Reply:
[818cb40](https://github.com/vivid-planet/comet-brevo-module/pull/52/commits/818cb40799ed90aa33cfed1b9fd1c21f3dc1042e)

Reply:
I don't think that's what @thomasdax98 was suggesting. Having a query arg `targetGroupId` is perfectly fine. I was thinking about a `manuallyAssignedBrevoContacts` query

Reply:
> I don't think that's what @thomasdax98 was suggesting. Having a query arg `targetGroupId` is perfectly fine. I was thinking about a `manuallyAssignedBrevoContacts` query

Yes, that's exactly what I meant

Reply:
Renamed `brevoContactsInTargetGroup` to `manuallyAssignedBrevoContacts` and removed the `onlyShowManuallyAssignedContacts` argument.
Readded targetGroupId to the "normal" `brevoContacts` query

[4aaf79c](https://github.com/vivid-planet/comet-brevo-module/pull/52/commits/4aaf79c26659507fcd88f085e6e07f08fe7b34e8)